cmd/sidecars/network-passt-binding/domain/configurator_test.go

```diff
@@ -347,6 +349,45 @@ var _ = Describe("pod network configurator", func() {
 
 			Expect(testMutator.Mutate(mutatedDomSpec)).To(Equal(mutatedDomSpec))
 		})
+	})
+	Context("should define memoryBacking for vhost-user", func() {
```

Comment:
nit: Please consider squashing the two commits.
I think it gives additional context when reviewing the changes to unit tests alongside the change to business logic.